about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Page</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Gentium+Plus:wght@700&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="main-nav bg-secondary">
+        <div class="first-container">
+            <ul class="main-nav-ul float-right">
+                <li class="main-nav-i"><i class="bi bi-facebook"></i></li>
+                <li class="main-nav-i"><i class="bi bi-twitter"></i></li>
+                <li class="main-nav-i">English</li>
+                <li class="main-nav-i">My Page</li>
+                <li class="main-nav-i">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <header class="second-nav" id="first-header">
+        <div class="first-container">
+            <i class="fa-solid fa-bars" id="menu-image></i>
+            <div class="navigation">
+                <a href="index.html"><img class="navigation-logo" src="assets/musicFestivalLogo.png" alt="Festival"></a>
+                <ul class="second-nav-ul float-right">
+                    <li><a class="second-nav-i" href="index.html">Home</a></li>
+                    <li><a class="second-nav-i" href="">Program</a></li>
+                    <li><a class="second-nav-i" href="">Join</a></li>
+                    <li><a class="second-nav-i" href="">Sponsor</a></li>
+                    <li><a class="second-nav-i" href="">News</a></li>
+                    <li><a href="#" class="mt-4 p-3" id="cc-btn">CC Campaign</a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <nav class="mobile-nav">
+        <span class="cross">x</span>
+        <ul class="mobile-nav-ul">
+            <li><a class="mobile-nav-i" href="index.html">Home</a></li>
+            <li><a class="mobile-nav-i" href="">Program</a></li>
+            <li><a class="mobile-nav-i" href="">Join</a></li>
+            <li><a class="mobile-nav-i" href="">Sponsor</a></li>
+            <li><a class="mobile-nav-i" href="">News</a></li>
+            <li><a href="#" class="mt-4 p-3" id="cc-btn">CC Campaign</a></li>
+        </ul>
```

Comment:
- [ ] It seems like your about page is not properly rendered, Looking at the screenshot below, you can see that text, images tends to overflow and overlap into different sections, I suggest you adjust the stylings appriopratly to look like the given design:

- - Your Design:
![Screenshot from 2023-05-25 12-04-03](https://github.com/Kgomotso196/FirstCapstoneProject/assets/107632780/5a39dbb8-216f-4541-a978-d89608614e58)


- - Provided Design:
![Screenshot from 2023-05-25 12-04-34](https://github.com/Kgomotso196/FirstCapstoneProject/assets/107632780/67c87744-4c00-4484-96c4-c785b0e5f5d8)